tests/migrate/fix_course_coach.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/* For licensing terms, see /license.txt */
+/**
+ * Set course coach if there is not assigned already one 
```

Comment:
:-)
"Set course coach if there is not already one assigned" sounds much more English